core/templates/services/context.service.spec.ts

```diff
@@ -266,5 +315,15 @@ describe('Context service', () => {
       expect(ecs.getPageContext()).toBe('other');
     }
     );
+
+    it('should retrive exploration preview mode', () => {
+      expect(ecs.getEditorTabContext()).toBe('preview');
+    });
```

Comment:
Where are we setting the context to be preview mode? Also, please include a precheck.

Reply:
Can you elaborate on pre check over here ?

Reply:
Done!